### PR TITLE
Project : Facial Expression Recognition with PyTorch Issue #697

### PR DESCRIPTION
**Pull request by revanth1718**
✔️ Involved training neural networks to accurately classify facial expressions using annotated facial image datasets, models were constructed and trained with PyTorch's deep learning capabilities.
✔️ Project was employed by convolutional neural networks (CNNs).
✔️ The success of the project was gauged through the model's proficiency in accurately identifying expressions on unseen data, with potential applications ranging from emotion-aware interfaces to behavioral analysis in psychology and market research.

Screenshot (550)

I am Participating in GSSOC'24. Please Label this Project as Level 3.

### CODE REVIEW

**Comment by invigorzz313** [comment]
First of all, please avoid committing from the master branch of your forked/cloned repo.
Create a new branch before creating a PR.
